app/controllers/poms_controller.rb

```diff
@@ -19,6 +22,22 @@ def show
 
   def edit; end
 
+  def my_caseload
+    user = Nomis::Elite2::Api.fetch_nomis_user_details(current_user).data
```

Comment:
Probably in a service like the others, and then that call wrapped in a method called user?